src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -935,25 +938,31 @@ public void testInspectContainerWithSecurityOpts() throws Exception {
 
   @Test
   public void testContainerWithHostConfig() throws Exception {
```

Comment:
For reference, Docker API 1.15 is the last version where `/containers/(id)/start` took a HostConfig JSON in the POST data. https://docs.docker.com/reference/api/docker_remote_api_v1.15/#start-a-container